### PR TITLE
desired behavior doc

### PR DESCRIPTION
**Pull request by usepowershell**
@mwrock I think, given our experiments, it'd be good to clarify what we think the expected behavior with respect to framework and BCL shipped with habitat vs the OS.

One of the challenges I see is making 4.0 apps disregard 4.5.1 assemblies.

Signed-off-by: Steven Murawski steven.murawski@gmail.com

### CODE REVIEW

File: desired_behavior.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+### CLR is Present / FRAMEWORK is Present
+
+Behavior:
+
+* Application uses System version of CLR
+* Application uses System version of FRAMEWORK 
+
+### CLR is Present / FRAMEWORK is Not Present
+
+Not Possible
```
Comment (mwrock) [inline desired_behavior.md:103]: I believe this is possible. The 4 CLR can exist with the .net 4.6 which is the default setup on server 2016. In which case we'd use the system CLR and Habitat's framework bits.

**Comment by mwrock** [comment]
With the exception of my comment above I'm 👍

File: desired_behavior.md (hunk 1 above)
Comment (usepowershell) [reply desired_behavior.md:103]: I can't think of an install option that would have the 2.0 CLR without the 2.0 Framework.

Comment (mwrock) [reply desired_behavior.md:103]: Me neither. Just with 4.

**Pull request merged**